app/src/main/java/chat/rocket/android/authentication/signup/ui/SignupFragment.kt

```diff
@@ -95,6 +111,41 @@ class SignupFragment : Fragment(), SignupView {
         }
     }
 
+    override fun saveSmartLockCredentials(loginCredential: Credential?) {
+        credentialsToBeSaved = loginCredential
+        if (googleApiClient!!.isConnected) {
+            saveCredentials()
+        }
+    }
+
+    override fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+        if (requestCode == SAVE_CREDENTIALS) {
+            if (resultCode == RESULT_OK) {
+                Toast.makeText(context, "Credentials saved successfully", Toast.LENGTH_SHORT).show()
```

Comment:
Hard coded string. We always should avoid that!. 

Please, add that string in the string.xml file (remember to adding the entries for another translations - Portuguese, Hind and so on). Note: You don't need to translate everything, just add a TODO (check how looking at some strings in that .xml files).